devops/windows/base-docker-wsl/helper/log/logs-jempi-api-kc.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -u
+
+source ../../0-conf.env
+docker service logs --follow --raw ${STACK_NAME}_api-kc
```

Comment:
Should we not remove the java app logs commands. Here and below as well?

Reply:
this must stay --- when api-kc is running, this shows the loads from this app